assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -54,6 +50,11 @@ const sizeFromWidth = (
   if (respectSetWidth) {
     return { width, height: width / PAGE_RATIO };
   }
+
+  if (bp === 'desktop') {
+    availableContainerSpace -= DASHBOARD_LEFT_NAV_WIDTH;
```

Comment:
turns out it's better if we base this math on the straight breakpoint not the inner width because the inner width changes based on if the wordpress menu is collapsed wheras the dashboard nav menu is based on the full viewport width. Doing this instead of `getTrueInnerWidth` helps _alot_